sql-cli/sql_cli/project.py

```diff
@@ -80,50 +87,71 @@ def data_dir(self) -> Path:
         return self._data_dir
 
     @property
-    def airflow_config(self) -> dict[str, Any]:
+    def environments_list(self) -> list[str]:
+        """
+        Return list with the names of the existing environments in the current project.
+
+        :returns: A list with strings representing the existing environments.
+        """
+        config_dir = self.directory / "config"
+        return [subpath.name for subpath in config_dir.iterdir() if subpath.is_dir()]
+
+    def get_env_airflow_config(self, env: str) -> dict[str, Any]:
         """
         Retrieve the Airflow configuration for the currently set environment.
 
         :returns: A Python dictionary containing the Airflow configuration.
         """
-        filename = self.airflow_home / "airflow.cfg"
+        airflow_home = self.get_env_airflow_home(env)
+        filename = airflow_home / "airflow.cfg"
         parser = ConfigParser()
         parser.read(filename)
         return {section: dict(parser.items(section)) for section in parser.sections()}
 
     def _initialise_global_config(self) -> None:
         """
-        Initialises global config file that includes configuration to be shared across environments including the
+        Initialises global config YAML file that includes configuration to be shared across environments including the
         airflow config.
         """
-        config = Config(project_dir=self.directory)
-        global_env_filepath = config.get_global_config_filepath()
+        config = Config(environment=DEFAULT_ENVIRONMENT, project_dir=self.directory)
```

Comment:
```suggestion
        config = Config(project_dir=self.directory)
```
Why not just this since Config now already has environment = DEFAULT_ENVIRONMENT as the default value?

Reply:
I cleaned it up - by the time I made this change, that was not the default :)